src/Aspire.Hosting/Publishing/ManifestPublishingContext.cs

```diff
@@ -358,7 +358,9 @@ public void WriteBindings(IResource resource)
             // 2. Default endpoints added by the framework
             // 3. Explicitly added endpoints
             // But wherever they come from, we treat the first one as Default, for each scheme.
```

Comment:
Does this comment need to updated to reflect that this behavior is only for http/https?

Reply:
Yes, and also that it only applies to project resources. e.g.

```c#
// This is used to determine if an http/https endpoint on a Project Resource should be treated as the Default endpoint
```